libcontainer/init_linux.go

```diff
@@ -440,35 +440,62 @@ func syncParentSeccomp(pipe *os.File, seccompFd *os.File) error {
 	return readSync(pipe, procSeccompDone)
 }
 
-// setupUser changes the groups, gid, and uid for the user inside the container
-func setupUser(config *initConfig) error {
-	// Set up defaults.
-	defaultExecUser := user.ExecUser{
-		Uid:  0,
-		Gid:  0,
-		Home: "/",
-	}
-
-	passwdPath, err := user.GetPasswdPath()
+func getExecUser(ustr string) (*user.ExecUser, error) {
```

Comment:
`str` looks to be referring to the type (which should already be clear from the signature); if we want a descriptive name, perhaps something like `userAndGroup` would work